packages/vm/src/buildBlock.ts

```diff
@@ -316,6 +320,14 @@ export class BlockBuilder {
       blobGasUsed = this.blobGasUsed
     }
 
+    let requests
+    let requestsRoot
+    if (this.vm.common.isActivatedEIP(7685)) {
+      const requests = await accumulateRequests(this.vm)
```

Comment:
Doesn't this `const requests` now flag the `let requests` above so the requests are always undefined?

Reply:
Good catch.  Thanks!